src/events/pollVotationResults.js

```diff
@@ -1,75 +1,107 @@
-const { Events } = require('discord.js');
+const { Events, roleMention, userMention } = require('discord.js');
 const { VOTE_POINTS } = require('../config');
 const { translateLanguage } = require('../languages');
 
 const tagIds = VOTE_POINTS.TAG_IDS;
 
+const getPointType = (questionPart) => {
+  const isBoostedPoint = questionPart.includes('boosted');
+  return isBoostedPoint ? tagIds.boostedPointTagId : tagIds.addPointTagId;
+};
+
+const parseQuestionInput = (questionFields) => {
+  const questionField = questionFields.find(
+    (field) => field.name === 'poll_question_text'
+  );
+
+  if (!questionField) {
+    return null;
+  }
+
+  const parts = (questionField.value || '')
+    .split('|')
+    .map((part) => part.trim());
+
+  if (parts.length < 2) {
+    return;
+  }
```

Comment:
same idea of throw an error but here, this error could be something like poll title doesn't have user ID. 

You can catch it out of the function if don't want handle it, but that way I think is easy to understand why you stopped the function execution

Reply:
Here it could say:

```js
throw new Error("Not found pointType and userId on poll description");
```